### PR TITLE
Update chinese.xml

### PR DESCRIPTION
**Pull request by danielkwan0516**
- 7.5.8 → 7.6
- Minor Amendments

### CODE REVIEW

File: PowerEditor/installer/nativeLang/chinese.xml [hunk 1]
```diff
@@ -177,15 +177,15 @@
...
+                    <Item id="43013" name="在檔案中尋找"/>
                     <Item id="43014" name="快速搜尋下一個"/>
                     <Item id="43015" name="快速搜尋上一個"/>
                     <Item id="43018" name="剪下標記行"/>
                     <Item id="43019" name="複製標記行"/>
                     <Item id="43020" name="貼上至標記行"/>
                     <Item id="43021" name="移除標記行"/>
                     <Item id="43051" name="移除未標記行"/>
-                    <Item id="43052" name="搜尋在範圍內的字元..."/>
+                    <Item id="43052" name="以字元編號搜尋字元..."/>
```
Comment (donho) [inline PowerEditor/installer/nativeLang/chinese.xml]: Good one

File: PowerEditor/installer/nativeLang/chinese.xml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <NotepadPlus>
-    <Native-Langue name="繁體中文/正體中文" filename="chinese.xml" version="7.5.8">
+    <Native-Langue name="繁體中文/正體中文" filename="chinese.xml" version="7.6">
```
Comment (donho) [inline PowerEditor/installer/nativeLang/chinese.xml]: Now it's 7.6.2, please solve the conflict.

Comment (danielkwan0516) [reply PowerEditor/installer/nativeLang/chinese.xml]: Solved. It was 7.6 when this PR opened.

File: PowerEditor/installer/nativeLang/chinese.xml [hunk 3]
```diff
@@ -178,15 +178,15 @@
                     <Item id="43009" name="跳至對應括號"/>
                     <Item id="43010" name="找上一個"/>
                     <Item id="43011" name="遞增搜尋(&amp;I)..."/>
-                    <Item id="43013" name="搜尋目錄"/>
+                    <Item id="43013" name="在檔案中尋找"/>
```
[the remark this answers is not in the document]
Comment (danielkwan0516) [reply PowerEditor/installer/nativeLang/chinese.xml]: OK. It is changed in line 350 but here is forgotten.

**Comment by donho** [comment]
I've got this error message:
```
$ git checkout -b danielkwan0516-master master
fatal: A branch named 'danielkwan0516-master' already exists.
```
You should ALWAYS create a new branch for a new PR.

**Pull request closed without merging**